opensearch/src/test/java/org/opensearch/sql/opensearch/planner/DefaultImplementorTest.java

```diff
@@ -0,0 +1,149 @@
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.jupiter.api.Test;
+import org.opensearch.sql.ast.tree.RareTopN.CommandType;
+import org.opensearch.sql.ast.tree.Sort;
+import org.opensearch.sql.data.model.ExprBooleanValue;
+import org.opensearch.sql.data.type.ExprCoreType;
+import org.opensearch.sql.opensearch.expression.OpenSearchDSL;
+import org.opensearch.sql.expression.Expression;
+import org.opensearch.sql.expression.NamedExpression;
+import org.opensearch.sql.expression.ReferenceExpression;
+import org.opensearch.sql.expression.aggregation.AvgAggregator;
+import org.opensearch.sql.expression.aggregation.NamedAggregator;
+import org.opensearch.sql.planner.DefaultImplementor;
+import org.opensearch.sql.planner.logical.LogicalPlan;
+import org.opensearch.sql.planner.logical.LogicalPlanDSL;
+import org.opensearch.sql.planner.physical.PhysicalPlan;
+import org.opensearch.sql.planner.physical.PhysicalPlanDSL;
+
+
+class DefaultImplementorTest {
+
+  private final DefaultImplementor<Object> implementor = new DefaultImplementor<>();
+
+  @Test
+  public void visit_should_return_default_physical_operator() {
+    String indexName = "test";
+    NamedExpression include = named("age", ref("age", INTEGER));
+    ReferenceExpression exclude = ref("name", STRING);
+    ReferenceExpression dedupeField = ref("name", STRING);
+    Expression filterExpr = literal(ExprBooleanValue.of(true));
+    List<NamedExpression> groupByExprs = Arrays.asList(OpenSearchDSL.named("age", ref("age", INTEGER)));
+    List<Expression> aggExprs = Arrays.asList(ref("age", INTEGER));
+    ReferenceExpression rareTopNField = ref("age", INTEGER);
+    List<Expression> topByExprs = Arrays.asList(ref("age", INTEGER));
+    List<NamedAggregator> aggregators =
+        Arrays.asList(OpenSearchDSL.named("avg(age)", new AvgAggregator(aggExprs, ExprCoreType.DOUBLE)));
+    Map<ReferenceExpression, ReferenceExpression> mappings =
+        ImmutableMap.of(ref("name", STRING), ref("lastname", STRING));
+    Pair<ReferenceExpression, Expression> newEvalField =
+        ImmutablePair.of(ref("name1", STRING), ref("name", STRING));
+    Pair<Sort.SortOption, Expression> sortField =
+        ImmutablePair.of(Sort.SortOption.DEFAULT_ASC, ref("name1", STRING));
+    Integer limit = 1;
+    Integer offset = 1;
+    List<Map<String, ReferenceExpression>> nestedArgs =
+        List.of(
+            Map.of(
+                "field", new ReferenceExpression("message.info", STRING),
+                "path", new ReferenceExpression("message", STRING)));
+    List<NamedExpression> nestedProjectList =
+        List.of(
+            new NamedExpression("message.info", OpenSearchDSL.nested(OpenSearchDSL.ref("message.info", STRING)), null));
+    Set<String> nestedOperatorArgs = Set.of("message.info");
+    Map<String, List<String>> groupedFieldsByPath = Map.of("message", List.of("message.info"));
+
+    LogicalPlan plan =
+        project(
```

Comment:
No need to copy and test all things here. You can test opensearch specific feature only here.
Try `plan = nested(values(List.of()))`